meta-bases/scan-base/autobuild/defines

```diff
@@ -0,0 +1,9 @@
+PKGNAME=scan-base
+PKGSEC=Bases
+PKGDEP="foomatic gutenprint tesseract"
+PKGDES="Meta package for AOSC OS scanning support"
```

Comment:
I think "Meta package for image scanning applications/support" is better.

Reply:
This is better handled in a survey (which will be part of the PKGDES translation project).